main.css

```diff
@@ -258,6 +262,155 @@ li:hover{
 
 .heading-product{
     display: inline-block;
-    margin-left:10rem;
-    margin-bottom:3rem;
+    margin-left: 10rem;
+    margin-bottom: 3rem;
+    margin-top: 8rem;
```

Comment:
shorthand property of margin and padding in Top, Right, Bottom, and Left use that in that order